plugins/modules/redfish_command.py

```diff
@@ -216,6 +216,24 @@
       - Handle to check the status of an update in progress.
     type: str
     version_added: '6.1.0'
+  update_custom_oem_header:
+    required: false
+    description:
+      - Optional OEM header, sent as separate form-data for
+        the Multipart HTTP push update.
+      - The header shall start with "Oem" according to DMTF
+        Redfish spec 12.6.2.2.
+      - If set, then `update_custom_oem_params` is required too.
+    type: str
+    version_added: '10.1.0'
+  update_custom_oem_params:
+    required: false
+    description:
+      - Custom OEM properties for HTTP Multipart Push Updates.
+      - If set, then `update_custom_oem_header` is required too.
+      - The content will be sent as JSON with application/json MIME type.
```

Comment:
The Redfish Spec doesn't require this form to be JSON-encoded. Realistically if there are OEM JSON options, those should just be encoded in the existing "Oem" objects in the common JSON form. However, I can see that some vendors may not follow that appropriately.

Is there any chance to open this up to support different MIME types?

Reply:
I was thinking about supporting different MIME types too but wasn't sure first if it really makes sense. But I am with you. I'll implement it.

Reply:
Implemented with commit 0ff8b76. Thanks for the feedback!